### PR TITLE
Milestone 1: Re-create Awesome Books

### PR DESCRIPTION
**Pull request by leopiresgoss**
## Milestone 1: Re-create Awesome Books

In this milestone I:

- Re-created the [Awesome Books](https://github.com/VicPeralta/awesomeBooks) using ES6 modules
- Used ES6 syntax (classes, const and let variables, and object destructuring)
- Used Luxon library to get the current date and time to updated user's date
- Improved its responsiveness 
- Updated README

[Live Demo](https://leopiresgoss.github.io/awesome-books-restructured/)

### CODE REVIEW

File: modules/set-listeners/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { displayBooks } from '../render-books/index.js';
+import navToSection from './nav-to-section.js';
+
+export default function setListeners(gBookShelf) {
+  document.addEventListener('click', (e) => {
+    if (e.target.matches('.remove-book')) {
```
Comment (elmar8287) [inline modules/set-listeners/index.js]: - [ ] Please, consider using the arrow function syntax in the whole code. For example, here you can go on with 

```
const setListeners = (gBookShelf) => {
....
```
and in the end, you can export it by default, as:

`export default setListeners;`

File: modules/render-books/display-books.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export default function displayBooks(gBookShelf) {
```
Comment (elmar8287) [inline modules/render-books/display-books.js]: - [ ] Please, consider to use arrow function syntax 👍

File: modules/set-listeners/nav-to-section.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export default function navToSection(element, sectionName) {
```
Comment (elmar8287) [inline modules/set-listeners/nav-to-section.js]: - [ ] Please, consider to use arrow function syntax 👍

File: modules/update-date/index.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import setDate from './set-date.js';
+
+export default function updateDate() {
```
Comment (elmar8287) [inline modules/update-date/index.js]: - [ ] Please, consider to use arrow function syntax 👍

File: modules/update-date/ordinal-day.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+export default function ordinalDay(day) {
```
Comment (elmar8287) [inline modules/update-date/ordinal-day.js]: - [ ] Please, consider to use arrow function syntax 👍

File: modules/update-date/set-date.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import ordinalDay from './ordinal-day.js';
+
+export default function setDate() {
```
Comment (elmar8287) [inline modules/update-date/set-date.js]: - [ ] Please, consider to use arrow function syntax 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @leopiresgoss, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @leopiresgoss,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**